crmprtd/process.py

```diff
@@ -132,7 +161,23 @@ def main():
         args.log_conf, args.log_filename, args.error_email, args.log_level, "crmprtd"
     )
 
-    process(args.connection_string, args.sample_size, args.network, args.diag)
+    # Right now start date defaults to one month earlier and
+    # end date defaults to now, but not sure if there
```

Comment:
I'd agree that the default behaviour should be *no* date filtering... i.e. if given no input, the process script should accept and process all of the data that it is given.